packages/@okta/vuepress-site/docs/guides/terraform-mange-multiple-domains/main/index.md

```diff
@@ -0,0 +1,262 @@
+
+#### Update the issuer for any OpenID Connect (OIDC) apps
+
+Consider changing the issuer URL for your OpenID Connect apps that use the built-in org authorization server. See the provider for [details about the app_oauth resource](https://registry.terraform.io/providers/okta/okta/latest/docs/resources/app_oauth).
+
+Choose a setting for the `issuer_mode` attribute. The most common setting is `DYNAMIC`. This uses either the organization URL or the custom domain URL depending on the domain of the system that requests the connection. The `CUSTOM_URL` setting enables a custom URL, but only when there's one brand. See [Custom URL is not Showing in the Issuer in the Custom Authorization Server Settings](https://support.okta.com/help/s/article/custom-issuer-url-is-not-showing-in-the-custom-server-settings?language=en_US).
+
+For example:
+
+```hcl
+resource "okta_app_oauth" "MyApp" {
+  label                      = "MyApp"
+  type                       = "web"
+  redirect_uris              = ["http://MyApp.example.com/authorization-code/callback"]
+  response_types             = ["code"]
+  grant_types                = ["authorization_code", "refresh_token", "client_credentials"]
+  token_endpoint_auth_method = "private_key_jwt"
+
+  implicit_assignment = true
+  login_mode = "DISABLED"
+  login_uri = "http://MyApp.example.com"
+
+  …
+
+  issuer_mode = CUSTOM_URL
+}
+```
+
+#### Update the issuer for any custom authorization servers
+
+Existing existing [custom authorization servers](/docs/concepts/auth-servers/) continue to use the Okta org URL unless you change it. All new custom authorization servers use the custom domain by default.
+
+Update existing custom authorization servers to return the custom domain as the issuer value. The choices for the issuer_mode attribute are the same as those for [OIDC apps](#update-the-issuer-for-any-openid-connect-oidc-apps).
+
+```hcl
+resource "okta_auth_server" "example" {
+  audiences   = ["api://example"]
+  description = "My Example Auth Server"
+  name        = "example"
+  issuer_mode = "CUSTOM_URL"
+  status      = "ACTIVE"
+}
+```
+
+#### Update external applications that connect to Okta
+
+If you have external apps that use Okta endpoints with the uncustomized URL domain, update them to use the custom URL domain.
+
+## Customize the look and the behavior of your brands
+
+You can customize the look and the behavior of your brands. There are two main customization resources available to you in the Okta Terraform provider:
+
+* **Brand**: A high-level resource that contains some settings, such as a custom privacy policy, is associated with a custom domain. See the [Brands concept article](/docs/concepts/brands/).
+
+* **Theme: **Settings that change the appearance of the user sign-in widget, such as the logo, background image, colors, and how the colors are applied. Each brand has only one theme, which Okta creates automatically when the brand is created.
```

Comment:
Bold isn't applied here without a fix for some reason
```suggestion
* **Theme**: Settings that change the appearance of the user sign-in widget, such as the logo, background image, colors, and how the colors are applied. Each brand has only one theme, which Okta creates automatically when the brand is created.
```

Reply:
The space between the colon and bold indicator causes issues. Fixed in next commit